solutions/devicesimulation-nohub/single-vm/docker-compose.yml

```diff
@@ -8,20 +8,23 @@ services:
     depends_on:
```

Comment:
Have you tested this file? e.g. deploy DS, get into the VM, and pull your msvc images and this docker compose file

Reply:
Verified on a VM